dataquality/clients/api.py

```diff
@@ -815,3 +815,11 @@ def set_metric_for_run(self, project_id: UUID4, run_id: UUID4, data: Dict) -> Di
             ),
             body=data,
         )
+
+    def get_healthcheck_dq(self) -> Dict:
+        return self.make_request(
+            RequestType.GET, url=f"{config.api_url}/{Route.healthcheck_dq}"
+        )
+
+    def get_bucket_names(self) -> Dict:
+        return self.get_healthcheck_dq()["bucket_names"]
```

Comment:
should we do a `.get()` here?
```suggestion
        return self.get_healthcheck_dq().get("bucket_names", {})
```